profiles/targets/desktop/plasma/package.use

```diff
@@ -29,13 +29,13 @@ dev-python/pyside6 webchannel
 # Required by sys-apps/flatpak, a dependency of kde-plasma/discover
 dev-util/ostree curl
 
-# Required by net-im/neochat:5
-dev-qt/qtmultimedia:5 gstreamer
+# Required by net-im/neochat
+dev-qt/qtmultimedia gstreamer
```

Comment:
Just noticed this, is dropping :5 really needed? Qt6 ebuilds depending on `[gstreamer]` are likely a relic from when the qtmultimedia:6 ebuild did not have: `REQUIRED_USE="|| ( ffmpeg gstreamer )"` -- I don't *think* anything specifically should need gstreamer and ffmpeg does the job.

Reply:
(the default is ffmpeg which is the favored backend upstream last I know of)

Reply:
..on that note, arguably the qt5 ebuild should probably not even have a IUSE=gstreamer, having qtmultimeda with no backend is kind of silly. Albeit at this point not much reason to bother changing that up.

Reply:
this was added in commit 0d8a8e5f740e5021cf80d4fbada4e97bc482de94, it may or may not be still necessary

https://bugs.gentoo.org/914004

Other revdeps currently are:
- dev-libs/kirigami-addons
- kde-plasma/kwin
- media-sound/elisa

(and waiting to be ported, games-kids/gcompris)

Reply:
@gerion0 would you like and be able to test this? (testing neochat's video capability after forcibly building `qtmultimedia:6[ffmpeg,-gstreamer]`.

Reply:
Yeah for slot 5 it makes sense for neochat and others, without a backend you can't decode anything and ffmpeg wasn't an option.

Reply:
> @gerion0 would you like and be able to test this? (testing neochat's video capability after forcibly building `qtmultimedia:6[ffmpeg,-gstreamer]`.

Neochat plays videos just fine with qtmultimedia:6[ffmpeg,-gstreamer].

Reply:
Thanks!